snippets/gift-card-recipient-form.liquid

```diff
@@ -0,0 +1,164 @@
+{% comment %}
+  Renders gift card recipient form.
+  Accepts:
+  - product: {Object} product object.
+  - form: {Object} the product form object.
+  - section: {Object} section to which this snippet belongs.
+
+  Usage:
+  {% render 'gift-card-recipient-form', product: product, form: form, section: section %}
+{% endcomment %}
+
+<div class="customer">
+  {%- assign gift_card_recipient_control_flag = "properties[__shopify_send_gift_card_to_recipient]" -%}
+  <script src="{{ 'recipient-form.js' | asset_url }}" defer="defer"></script>
+  <recipient-form class="recipient-form"
+    data-section-id="{{ section.id }}"
+    data-product-variant-id="{{ product.selected_or_first_available_variant.id }}"
+  >
+    <input
+      id="Recipient-Checkbox-{{ section.id }}"
+      type="checkbox"
+      name="{{ gift_card_recipient_control_flag }}"
+      disabled
+      unchecked
+    >
+    <label class="recipient-checkbox" for="Recipient-Checkbox-{{ section.id }}">
+      <svg
+        width="1.6rem"
+        height="1.6rem"
+        viewBox="0 0 16 16"
+        aria-hidden="true"
+        focusable="false"
+      >
+        <rect width="16" height="16" stroke="currentColor" fill="none" stroke-width="1.75"></rect>
+      </svg>
+      {%- render 'icon-checkmark' -%}
```

Comment:
This is where the stroke seemed different. We could remove this `render` and use the same code as in the facets.liquid file: 
```html
<svg
  aria-hidden="true"
  class="icon icon-checkmark"
  width="1.1rem"
  height="0.7rem"
  viewBox="0 0 11 7"
  fill="none"
  xmlns="http://www.w3.org/2000/svg"
>
  <path d="M1.5 3.5L2.83333 4.75L4.16667 6L9.5 1" stroke="currentColor" stroke-width="1.75" stroke-linecap="round" stroke-linejoin="round" />
</svg>
```

But some styling will need to be tweaked as well as that svg won't be aligned properly when replacing the render. 
Seems like using those values could do the trick (removing the original height added to it):
```css 
    position: absolute;
    left: 0.3rem;
    z-index: 5;
    top: 0.4rem;
```

Reply:
Ill have a look at it, it should be a quick update 👍